docs/source/advanced.rst

```diff
@@ -128,6 +128,43 @@ offer an easy way to compile and tune compilation parameters. Examples:
 In all cases, returning an un-compiled model is equivalent to
 calling ``model.compile(**compile_kwargs)`` within ``model_build_fn``.
 
+.. _loss-selection:
+
+Loss selection
+++++++++++++++
+
+If you do not explicitly define a loss, SciKeras attempts to find a loss
+that matches the type of target (see :py:func:`sklearn.utils.multiclass.type_of_target`).
+
+For guidance selecting losses in Keras, please see Jason Brownlee's
+excellent article `How to Choose Loss Functions When Training Deep Learning Neural Networks`_
+as well as `Keras Losses docs`_.
+
+Default losses are selected as follows:
+
+Classification
+..............
```

Comment:
I think this section could use some use examples, and clarification of what "output" and "encoding" mean.

Reply:
> In this context, `outputs` refers to the number of things you are predicting (for example, you could predict just `color`, in which case you have 1 output, or you might predict `color` and `is_tshirt`, in which case you have 2 outputs). Encoding refers to the representation of the target data. Generally, you will see data encoded as labels (`[1, 2, 3]` or `["red", "green", "blue"]`) or one-hot encoded. See [one-hot on Wikipedia](https://en.wikipedia.org/wiki/One-hot) for more details.